osu.Game/Overlays/Mods/ModButton.cs

```diff
@@ -151,8 +151,8 @@ public Mod Mod
         [BackgroundDependencyLoader]
         private void load(AudioManager audio)
         {
-            sampleOn = audio.Sample.Get(@"Checkbox/check-on");
-            sampleOff = audio.Sample.Get(@"Checkbox/check-off");
+            sampleOn = audio.Sample.Get(@"UI/check-on");

```

Comment:
Like `OsuColour` sounds most reasonable to me. I'd like to hear @peppy 's opinion on this, too. Something like `OsuAudio` and `OsuTextures` or just a single class `OsuResources`.